ci/get-unifi.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+OS_TYPE="$(uname -s | tr A-Z a-z)"
+ if [[ $(uname -m) == 'x86_64' ]]; then
+    CPU_ARCH='amd64'
+  else
+    CPU_ARCH="$(uname -m)"
+ fi
```

Comment:
```suggestion
if [[ $(uname -m) == 'x86_64' ]]; then
  CPU_ARCH='amd64'
else
  CPU_ARCH="$(uname -m)"
fi
```
Unnecessary indentation